pnc/src/main/java/org/jboss/pnc/bacon/pnc/common/ParameterChecker.java

```diff
@@ -39,7 +39,7 @@ public static void checkRebuildModeOption(String rebuildMode) throws FatalExcept
             for (RebuildMode mode : RebuildMode.values()) {
                 log.error(mode.toString());
             }
-            throw new FatalException();
+            throw new FatalException("Illegal option for rebuild mode");
```

Comment:
Would be great to have the value that is invalid printed somewhere (just a note, I know it's not changed in this PR).

Reply:
I'll add it to the exception.